### PR TITLE
feat : add writesonic as an organisation

### PR DESCRIPTION
**Pull request by Gupta-Anubhav12**
## Description

Add writesonic.com in the organizations list 

we today Open sourced https://gpt-router.writesonic.com/ built using Fastify and hence raising a PR to add writesonic.com as one of the adopters 
## Check List

- [x] I have read the [Contributing Guidelines](./CONTRIBUTING.md)

### CODE REVIEW

File: static/data/organisations.yml [hunk 1]
```diff
@@ -313,3 +313,8 @@
 - name: 'Investorean'
   image: 'investorean.svg'
   link: 'https://investorean.com'
+
+- name: 'Writesonic'
+  image: 'writesonic.svg'
+  link: 'https://writesonic.com'
+
```
Comment (Eomm) [inline static/data/organisations.yml]: this should fix the linter

```suggestion
```

Comment (Gupta-Anubhav12) [reply static/data/organisations.yml]: fixed @Eomm

**Review by Eomm — COMMENTED** [review]
It does not work well in light mode:

image

Is it ok for you?

**Comment by Uzlopak** [comment]
Closing due to inactivity

**Pull request closed without merging**